usd/src/usd_parser/usd2sdf_TEST.cc

```diff
@@ -99,6 +99,7 @@ TEST(check_cmd, IGN_UTILS_TEST_DISABLED_ON_WIN32(SDF))
     EXPECT_DOUBLE_EQ(0.0, world->Gravity()[1]);
     EXPECT_DOUBLE_EQ(-0.098, world->Gravity()[2]);
 
+    EXPECT_EQ(4u, world->Plugins().size());
```

Comment:
can you check the names of the plugins that exist as well? That way, we can verify that we have the `sensors`, `physics`, `sceneBroadcaster`, and `userCommands` plugins in the sdf file.